src/components/v5/shared/UserPopover/partials/UserInfo.tsx

```diff
@@ -66,79 +66,88 @@ const UserInfo: FC<UserInfoProps> = ({
           </>
         )}
       </div>
-      {aboutDescriptionText && (
-        <>
-          <TitleLabel
-            className={clsx('mb-2', {
-              'px-6': isTopContributorType,
-            })}
-            text={formatText({ id: 'userInfo.about.section' })}
-          />
-          <p
-            className={clsx('text-md text-gray-600', {
-              'px-6': isTopContributorType,
+      <div
+        className={clsx({
+          'pt-6': aboutDescriptionText || domains?.length,
```

Comment:
would it be possible for us to put the padding to the `aboutDescriptionText` and/or the domains section? so we don't need conditional classNames and we will just get spacing from those components?

Reply:
@bassgeta done :)